apps/store/src/features/memberArea/PaymentsSection/PaymentConfig.tsx

```diff
@@ -0,0 +1,75 @@
+import { datadogRum } from '@datadog/browser-rum'
+import styled from '@emotion/styled'
+import { useState } from 'react'
+import { Button, Dialog, theme } from 'ui'
+import { useMemberAreaMemberInfoQuery, useTrustlyInitMutation } from '@/services/apollo/generated'
+import { useAppErrorHandleContext } from '@/services/appErrors/AppErrorContext'
+import { getTrustlyInitMutationVariables } from '@/services/trustly/createTrustlyUrl'
+import { TrustlyIframe } from '@/services/trustly/TrustlyIframe'
+import { useCurrentLocale } from '@/utils/l10n/useCurrentLocale'
+
+type State = { type: 'IDLE' } | { type: 'READY'; trustlyUrl: string }
+
+export const PaymentConfig = ({ startButtonText }: { startButtonText: string }) => {
+  const { showError } = useAppErrorHandleContext()
+  const { routingLocale } = useCurrentLocale()
+  const [createTrustlyUrl, result] = useTrustlyInitMutation({
+    onError: showError,
+    onCompleted(data) {
+      const trustlyUrl = data.registerDirectDebit2.url
+      setState({ type: 'READY', trustlyUrl })
+    },
+  })
+  const [state, setState] = useState<State>({ type: 'IDLE' })
+
+  const handleClickStart = () => {
+    datadogRum.addAction('Payment Config Start')
+    createTrustlyUrl({ variables: getTrustlyInitMutationVariables(routingLocale) })
+  }
+
+  const handleClose = () => {
+    datadogRum.addAction('Payment Config Close')
+    setState({ type: 'IDLE' })
+  }
+
+  const { refetch } = useMemberAreaMemberInfoQuery()
+  const handleSuccess = async () => {
+    datadogRum.addAction('Payment Config Success')
+    await refetch()
+    setState({ type: 'IDLE' })
+  }
+
+  const handleFail = () => setState({ type: 'IDLE' })
+
+  return (
+    <>
+      <Button loading={result.loading} onClick={handleClickStart}>
+        {startButtonText}
+      </Button>
+
+      <Dialog.Root open={state.type === 'READY'} onOpenChange={handleClose}>
+        <DialogIframeContent onClose={handleClose}>
+          <DialogIframeWindow>
+            {state.type === 'READY' && (
+              <TrustlyIframe url={state.trustlyUrl} onSuccess={handleSuccess} onFail={handleFail} />
+            )}
+          </DialogIframeWindow>
+        </DialogIframeContent>
+      </Dialog.Root>
+    </>
+  )
+}
+
+const DialogIframeContent = styled(Dialog.Content)({
+  height: '100%',
+  maxHeight: 800,
+  width: '100%',
+  maxWidth: 600,
+  padding: theme.space.xs,
+})
```

Comment:
Looks like the iframe isn't centered :) 
<img width="1720" alt="Screenshot 2023-10-17 at 14 34 07" src="https://github.com/HedvigInsurance/racoon/assets/6661511/abcdf0a4-0b16-41fe-9aac-2020925947a0">



```suggestion
  marginInline: 'auto',
})
```

Reply:
so strange, must be something up with dev env so it didn't sync my styles correctly